src/main/java/com/agilethought/atsceapi/controller/UserController.java

```diff
@@ -1,20 +1,33 @@
-package com.agilethought.atsceapi.controller;
-
-import com.agilethought.atsceapi.model.LoginData;
-import com.agilethought.atsceapi.model.User;
-import com.agilethought.atsceapi.service.UserService;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.*;
-
-@RestController
-@RequestMapping(value = "/at-sce-api/api/v1")

```

Comment:
You deleted `@RequestMapping(value = "/at-sce-api/api/v1")` that is required to work properly